update version numbers + Text updates (#5463)
umbraco/UmbracoDocs: new-backoffice/tutorials/creating-a-custom-dashboard/adding-functionality-to-the-dashboard.md

The steps we will go through in this part are:

1. [Contexts](adding-functionality-to-the-dashboard.md#1.-contexts)
2. [Getting data from the server](adding-functionality-to-the-dashboard.md#2.-getting-data-from-the-server)

## Step 1: Contexts

> Umbraco has a large selection of contexts that you can use in your custom Property Editors and Dashboards. For this example, it would be nice to welcome the editor by name. To achieve this we can make use of the Umbraco Contexts.

Reviewer: The example is about welcoming the editor by name so things should be kept simple and on point 💡 
```suggestion
Umbraco has a large selection of contexts that you can use in your custom Property Editors and Dashboards. For this example, we will welcome the editor by name. To achieve this we can make use of the Umbraco Contexts.
```
